frontend/src/components/custom-tools/prompt-card/PromptCardItems.jsx

```diff
@@ -94,27 +94,27 @@ function PromptCardItems({
   const [isIndexOpen, setIsIndexOpen] = useState(false);
   const { width: windowWidth } = useWindowDimensions();
   const componentWidth = windowWidth * 0.4;
-
+  const isNotSingleLlmProfile = llmProfiles.length > 1;
```

Comment:
@jagadeeswaran-zipstack Hope we have handled the null exception here for `llmProfiles` since we are getting it from store, and directly using it for `map` and other operations.

Reply:
Default value for `llmProfiles` is empty array. So no need to handle null exception here